src/Symfony/Component/Notifier/Bridge/Slack/README.md

```diff
@@ -1,7 +1,7 @@
 Slack Notifier
 ==============
 
-Provides Slack integration for Symfony Notifier.
+Provides [Slack](https://slack.com) integration for Symfony Notifier.
 
 Resources
```

Comment:
`DSN example` ?

Reply:
Damn missing, thanks ☺️ will take a look later

Reply:
updated

@malteschlueter I think this is the correct one for `5.1`

Reply:
Looks good for me :)